test/sharness/t0040-add-and-cat.sh

```diff
@@ -156,6 +156,15 @@ test_add_cat_file() {
     test_cmp expected actual
   '
 
+  test_expect_success "ipfs add --chunker rabin-36-512-1024 succeeds" '
+    ipfs add --chunker rabin-36-512-1024 mountdir/hello.txt >actual
+  '
+
+  test_expect_failure "ipfs add --chunker rabin-12-512-1024 failed" '
```

Comment:
FYI, `test_expect_failure` is actually poorly named. It's for testing known bugs, not testing things that *should* fail. This should actually just be `test_expect_code 1 add --chunker ...`